app.js

```diff
@@ -116,6 +116,7 @@ app.use(session({
 // CSRF middlewares
 app.use(duplicateTokenHandler);
 app.use(csurf());
+app.use(errorHandler);
```

Comment:
Error handlers should be moved to around line 188, place this after the 404 error handler and give it a better name like CsrfErrorHandler